examples/launcher/src/components/Engine.jsx

```diff
@@ -187,7 +187,7 @@ class Engine extends React.Component {
       const url = urlInput.value;
 
       window.postMessage({
-        method: 'open',
+        method: 'reload',
         url,
         d: null,
```

Comment:
What dimensionality should the new tab be? I'm guessing `3`.

Reply:
Perhaps there should be 2 separate "reload buttons":

- Close all tabs, and open a 2d layer 
- Close all layers, and open a 3d layer

one for d of `2` and one for `3`

Reply:
If the intent of this functionality is to be "like a browser" then we probably don't want any additional knobs here...

Reply:
![2d3dCapture](https://user-images.githubusercontent.com/29695350/57987886-3d3d2a80-7a4d-11e9-89ea-dc9d61f3ce0d.PNG)

It is mirroring the current options of opening a new 2d/3d layer^

Reply:
But what will the effect be on `openUrl` when no dimension is specified?

Reply:
`openUrl` defaults to d of `3` https://github.com/exokitxr/exokit/blob/studio/examples/studio.html#L608

Reply:
I see, so this is just confusing code then. Either way I'm not sure if the reload concept makes any sense, because it's not clear what the UX of it would be compared to the other cases.

It seems we're navigating a different abstraction here -- we are navigating an orchestration of realities, not a reality tab.

Maybe what we are looking for is the ability to edit the src of a reality tab?

Reply:
I think the right thing for this case is "Clear all layers", which would use this current PR with no opening of a new url.

Reply:
In general, a stupider way to navigate iframes might be best, then? Think the DOM view, except only reality tabs, with a clear button.

Reply:
Generally the most front-facing UI should be very stupid, the DOM view is even too hairy right now,

> DOM view, except only reality tabs, with a clear button. 
 
I think this should be it, with the ability to click into and *then* see/edit the full DOM/source

Reply:
+1 to that.

In that case we wouldn't need the reload option at all.

Reply:
removed in f9febc4